index.html

```diff
@@ -0,0 +1,338 @@
+                    <div class="col-md-12 col-3 align-self-center text-center ">
+                      <h5 class="text-magenta h5style3">
+                        Style Feed
+                      </h5>
+                    </div>
+                    <div class="col-md-12 col-7 pt-3 pl-3 align-self-center text-justify ">
+                      <p class="text-light h5style4">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit officia ea, aliquam eaque ipsam voluptate non illum, cum beatae tenetur, quas quia exercitationem nulla dolorem dolore dolor minima porro. Iste.                  
+                      </p>
+                    </div>
+                  </div>
+                </div>
+                <div class="col-12 col-md-3">
+                  <div class="row bg-light-transparent m-1  ">
+                    <div class="col-md-12 col-2 align-self-center text-center ">
+                      <h5 class="text-light h5style2">
+                        <i class="far fa-eye fa-2x"></i>
+                      </h5>
+                    </div>
+                    <div class="col-md-12 col-3 align-self-center text-center ">
+                      <h5 class="text-magenta h5style3">
+                        Visual Identity
+                      </h5>
+                    </div>
+                    <div class="col-md-12 col-7 pt-3 pl-3 align-self-center text-justify ">
+                      <p class="text-light h5style4">
+                        Lorem ipsum dolor sit amet consectetur adipisicing elit. Odit officia ea, aliquam eaque ipsam voluptate non illum, cum beatae tenetur, quas quia exercitationem nulla dolorem dolore dolor minima porro. Iste.
+                      </p>
+                    </div>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row d-md-none mt-4 mb-3">
+            <div class="col-1"></div>
+            <div class="col-10">
+              <a class="btn btn-lg w-100 bg-magenta text-center text-light p-2" href="/">
+                See the whole list of services
+              </a>
+            </div>
+            <div class="col-1"></div>
+          </div>
+          <div class="row m-4 d-none d-lg-block">
+            <div class="col text-center text-light divstyle">
+              <a class="text-uppercase text-light font-weight-bold" href="/">
+                See the whole list of services
+              </a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section id="compmembers" class="row text-lato">
+        <div class="col">
+            <div class="row m-4">
+              <div class="col-12 text-center">
+                <h5 class="text-dark h5style1">
+                  Company Members
+                </h5>
+              </div>
+              <hr class="col-12">
+            </div>
+            <div id="speakers-container" class="row">
+            </div>
```

Comment:
[OPTIONAL] I think you might've missed this button on the mobile view. I would recommend you do this exercise. Is a definitively interesting challenge and part of the original design. I'm referring to the `more` button. 

It would become a `less` button once the section is expanded.

![image](https://user-images.githubusercontent.com/79114233/142355210-a845d55c-1559-4dde-b676-c36a29c4dcd6.png)

Reply:
Thank you @D4nielJ for your feedback. Your observation is now implemented on commit 991c85d.